fuzzing/CMakeLists.txt

```diff
@@ -0,0 +1,41 @@
+cmake_minimum_required(VERSION 2.8)
+
+# I really don't get GNU Make, and I couldn't figure out how to get it
```

Comment:
Can you remove this comment please? It won't be relevant for readers after review is done.